src/main/java/core/basesyntax/strategy/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction fruit) {
+        Integer amountBefore = Storage.storage.get(fruit.getName());
+        Integer valueAfter = amountBefore + fruit.getAmount();
+        Storage.storage.put(fruit.getName(), valueAfter);

```

Comment:
you can use hashmap.merge in handlers if you wish, with Integer::sum function here and in supply